types/microrouter/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for microrouter 2.2
+// Project: https://github.com/pedronauck/micro-router#readme
+// Definitions by: Mathieu Dutour <https://github.com/mathieudutour>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node"/>
+
+import { IncomingMessage, ServerResponse, Server } from 'http';
+
+export type RequestHandler = (
+    req: IncomingMessage & {
+        params: {[key: string]: string}
+    },
+    res: ServerResponse
+) => any;
```

Comment:
So it's not `send` but `RequestHandler`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/micro/index.d.ts#L9

The thing is, it's not exactly the same type: there are `params` and `query` added to the `IncomingRequest`.

So I used the `RequestHandler` type from `micro` but defined a `AugmentedRequestHandler` with the `params` and `query`